utah-design-system-website/src/react/components/websiteContent/library/components/forms/DateInput/DateInputDocumentation.jsx

```diff
@@ -1,17 +1,57 @@
 /* eslint-disable max-len */
+import { CalendarInput } from '@utahdts/utah-design-system';
 import { Link } from 'react-router-dom';
+import { useImmer } from 'use-immer';
 import dateInputScreenshot from '../../../../../../../static/images/screenshots/components/dateInput/dateInput.jpg';
 import { LightBox } from '../../../../../lightbox/LightBox';
 import { pageUrls } from '../../../../../routing/pageUrls';
+import { SandboxExample } from '../../../../../sandbox/SandboxExample';
 import { StaticExample } from '../../../../../staticExamples/StaticExample';
+import { DateInputExampleCodeReact } from './DateInputExampleCodeReact';
+import { DateInputExampleProps } from './DateInputExampleProps';
+import { DateInputExampleRender } from './DateInputExampleRender';
 
 export function DateInputDocumentation() {
+  const [calendarDate, setCalendarDate] = useImmer(/** @type {string | null} */(null));
   return (
     <div className="documentation-content">
       <h1 id="h1-top">Date Input</h1>
       <p className="lead-in">The Date Input is a specialized input field that can present a date picker popup to the user.</p>
       <hr />
-      <h2 id="example">Example</h2>
+      <h2 id="section-example">Example</h2>
+
+      <div style={{ width: '200px' }}>
+        <CalendarInput
+          label="test calendar"
+          labelClassName="visually-hidden"
+          isDisabled={false}
+          onChange={setCalendarDate}
+          id="test-calendar-input"
+          showTodayButton
+          value={calendarDate}
+        />
+      </div>
+
+      <SandboxExample
+        defaultProps={{
+          className: '',
+          errorMessage: '',
+          hasCalendarPopup: true,
+          id: 'date-input-example-id',
+          isClearable: false,
+          isDisabled: false,
+          label: 'Enter a value',
```

Comment:
Change this to: "Enter a date" it very confusing what the user is supposed to enter on screen reader.